mkdocs.yml

```diff
@@ -57,6 +57,8 @@ nav:
       - Developing a protocol: protocol.md
       - Using Git effectively: git-workflow.md
       - Code reviews: code-reviews.md
+  - Building for OpenSAFELY:
```

Comment:
nit (non-blocking): I feel like it's possible to have a slightly more precise title here. Unhelpfully, I don't know what might be. "Building datasets for OpenSAFELY?" "Integrating with OpenSAFELY"?

It's fine to leave for now in this PR and change later.

Reply:
Yeah, I think this is a task for later, this PR is just about getting the backend docstrings in.